### DISCUSSION TITLE
[问题反馈] 所有小说均无法阅读

### DISCUSSION THREAD

**Discussion by xialibaixue** [xiaoyaocz/flutter_dmzj#151] [General]
![Screenshot_20230327_225220](https://user-images.githubusercontent.com/77116861/227977678-becd5b70-18a1-4ccd-baa8-05adf3b8aed9.jpg)
![Uploading Screenshot_20230327_225218.jpg…]()
![Screenshot_20230327_225353](https://user-images.githubusercontent.com/77116861/227977938-b0fb9771-569a-4dc0-9233-3d171fcf5211.jpg)

如图中所示，小说列表以及目录的加载没有异常，但是点进任何一个小说进行阅读均会提示网络异常。

**Comment by xialibaixue** [comment]
![Screenshot_20230327_225218](https://user-images.githubusercontent.com/77116861/227978819-903dc38f-c5fe-4d6b-b1a1-4d0f62a5bf2c.jpg)

**Comment by xiaoyaocz** [comment]
[更新2.0.5](https://github.com/xiaoyaocz/flutter_dmzj/releases/tag/v2.0.5-beta)
